src/arche/rules/result.py

```diff
@@ -80,6 +94,25 @@ def info(self):
     def warnings(self):
         return self.messages.get(Level.WARNING)
 
+    @property
+    def outcome(self) -> Outcome:
+        if not self._outcome:
+            if not self.messages:
+                self._outcome = Outcome.PASSED
+            elif Level.ERROR in self.messages.keys():
+                self._outcome = Outcome.FAILED
+            elif Level.WARNING in self.messages.keys():
+                self._outcome = Outcome.WARNING
+            elif self.stats:
+                self._outcome = Outcome.INFO
+            else:
+                self._outcome = Outcome.PASSED
+        return self._outcome
+
+    @outcome.setter
+    def outcome(self, outcome):
```

Comment:
Probably this is an internal behavior and shouldn't be set outside the class

Reply:
Sometimes we want to override outcome, that's why there's a property.

Reply:
> @simoess The last thing left - add a line to changes https://github.com/scrapinghub/arche/blob/jinja_report/CHANGES.md#037dev-work-in-progress. From the user perspective

Done

Reply:
@ejulio I agree with @manycoding  on this one.